server/app/services/export/ProgramJsonSampler.java

```diff
@@ -56,6 +56,7 @@ public CfJsonDocumentContext getSampleJson(ProgramDefinition programDefinition)
             // for those tests, otherwise to the actual program ID.
             .setLanguageTag(Locale.US.toLanguageTag())
             .setSubmitterType(SubmitterType.APPLICANT)
+            .setSubmissionStatus("CURRENT")
```

Comment:
This could then become `LifecycleStage.ACTIVE.getDescription()` based on my comment on moving the enum. Then we don't have to remember to update it in the future.